### PR TITLE
Cab homepage

### PR DESCRIPTION
**Pull request by calebbratton**
working data to flights and hotels. need to add details to tiles for each

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -3,6 +3,4 @@ AUTH0_CLIENT_SECRET
 AUTH0_DOMAIN
 AUTH0_CLIENT_ID
 AUTH0_CALLBACK_URL
-SESSION_SECRET
-AMADEUS_CLIENT_ID
-AMADEUS_CLIENT_SECRET
+SESSION_SECRET
```
Comment (Pix3lAssassin) [inline .env-example]: Should not remove these from .env-example

File: client/src/components/Events/EventTile.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
```
Comment (Pix3lAssassin) [inline client/src/components/Events/EventTile.jsx]: could use object destructuring for this (just cosmetic)

File: client/src/components/Flights/FlightController.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      });
+  };
+
+  const onFlightInfoChange = (input) => (e) => {
+    flightInfo[input] = e.target.value;
+  };
+
+  function setDate(startDate, endDate) {
+    flightInfo.departureDate = startDate;
+    flightInfo.returnDate = endDate;
```
Comment (Pix3lAssassin) [inline client/src/components/Flights/FlightController.jsx]: should use setFlightInfo() here

File: client/src/components/Flights/FlightTile.jsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    width: 128,
+    height: 128,
+  },
+  img: {
+    margin: 'auto',
+    display: 'block',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+}));
```
Comment (Pix3lAssassin) [inline client/src/components/Flights/FlightTile.jsx:28]: Could extract this to the flight styles file

File: client/src/components/Flights/FlightTile.jsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+const displaySegments = (itinerary, arriveDepart) => {
+  if (itinerary.segments.length === 1) {
+    let arrivalTime = itinerary.segments[0][arriveDepart].at;
+    arrivalTime = arrivalTime.split('T');
+    const date = arrivalTime[0];
+    const time = arrivalTime[1];
+    return `${itinerary.segments[0][arriveDepart].iataCode} @
+    ${time} ${date}`;
```
Comment (Pix3lAssassin) [inline client/src/components/Flights/FlightTile.jsx]: Should cast to date and extract time and date that way. Using this method will account for timezone information

File: client/src/components/Flights/FlightTile.jsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                  Standard license
+                </Typography>
+                <Typography variant="body2" gutterBottom>
+                  Departure:
+                  {' '}
+                  {displaySegments(flight.itineraries[0], 'departure')}
+                  <br />
+                  Arrival:
+                  {' '}
+                  {displaySegments(flight.itineraries[0], 'arrival')}
```
Comment (Pix3lAssassin) [inline client/src/components/Flights/FlightTile.jsx:67]: Could use string templating for this

File: client/src/index.jsx [hunk 7]
```diff
@@ -56,18 +55,16 @@ function App() {
...
+      <CssBaseline />
+      <Navigation handleLogin={handleLogin} user={user} userLogin={userLogin} />
+      <div className={classes.main}>
+        <Router>
+          <Switch>
+            <Route path="/"><Home /></Route>
+            <Route path="/userPro"><UserProfile user={user} /></Route>
+          </Switch>
+        </Router>
+      </div>
```
Comment (Pix3lAssassin) [inline client/src/index.jsx]: should add appcontext

File: client/src/components/Hotels/HotelController.jsx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        setHotels(response.data.data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  const onHotelInfoChange = (input) => (e) => {
+    hotelInfo[input] = e.target.value;
+  };
```
Comment (Pix3lAssassin) [inline client/src/components/Hotels/HotelController.jsx]: if this is state it should be updated using setHotelInfo

File: client/src/components/Hotels/Hotels.jsx [hunk 9]
```diff
@@ -1,30 +1,29 @@
...
+    cityCode: '',
+    checkInDate: '',
+    checkOutDate: '',
+    adults: null,
+
+  });
+  const [hotels, setHotels] = useState([]);
   const classes = useStyles();
+  useEffect(() => {
+    console.log(hotels);
```
Comment (Pix3lAssassin) [inline client/src/components/Hotels/Hotels.jsx]: should remove this console.log

File: client/src/components/Hotels/HotelTile.jsx [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+            </ButtonBase>
+          </Grid>
+          <Grid item xs={12} sm container>
+            <Grid item xs container direction="column" spacing={2}>
+              <Grid item xs>
+                <Typography gutterBottom variant="subtitle1">
+                  {hotel.hotel.name}
+                </Typography>
+                <Typography variant="body2" gutterBottom>
+                  {hotel.hotel.description ? hotel.hotel.description.text : <span>undefined</span>}
```
Comment (Pix3lAssassin) [inline client/src/components/Hotels/HotelTile.jsx]: Should replace undefined with something more user friendly

File: client/src/components/Hotels/HotelDatePicker.jsx [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+    if (day.length < 2) {
+      day = '0' + day;
+    }
+    return [year, month, day].join('-');
+  }
+  const onDateChange = (startDate, endDate) => {
+    const start = formatDate(startDate);
+    const end = formatDate(endDate);
+    hotelInfo.checkInDate = start;
+    hotelInfo.checkOutDate = end;
```
Comment (Pix3lAssassin) [inline client/src/components/Hotels/HotelDatePicker.jsx:24]: should use setHotelInfo home

**Review by mhhutton — APPROVED** [review]

**Pull request merged**